iterators.tex

```diff
@@ -623,31 +650,186 @@
 \textit{p}).
 \exitexample
 
-\indexlibrary{\idxcode{Iterator}}%
-\begin{codeblock}
+{\color{newclr}
+\rSec2[iterators.sizediterator]{Concept SizedIterator}
+\pnum
+The \tcode{SizedIterator} concept refines \tcode{Iterator} with the
+capability to determine the distance between two iterators that denote
+elements of the same range in constant time with the \tcode{-} operator.
+
+\indexlibrary{\idxcode{SizedIterator}}%
+\begin{itemdecl}
+  template <class>
+  constexpr bool disable_sized = false;
```

Comment:
IMO it's a problem that `disable_sized<I>` and `disable_sized<const I>` will yield different results if the user has specialized `disable_sized` on `I`. I'm also uncomfortable that the sentinel type isn't considered when using `disable_sized` to check the `SizedSentinel` concept.